sandbox/tests/unit tests/inputs/test_projectfilereader_configurationblocks.appleseed

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project format_revision="22">
+<project format_revision="23">
```

Comment:
You are two GSoC candidates adding a new project file migration step, they will conflict. Let's see who gets his PR merged first :)